### DISCUSSION TITLE
Save password with keyring package like pgcli

### DISCUSSION THREAD

**Discussion by khaitranhq** [dbcli/mycli#1173] [Q&A] [answered]
Hi everyone.
I'm very excited with the feature saving password using the `keyring` package of `pgcli`. Can I use this feature in `mycli`? I tried but it didn't work.

**Comment by rolandwalker** [comment, marked as answer]
Hi!  This was addressed in https://github.com/dbcli/mycli/pull/1499 and many followup PRs.
